index.md

```diff
@@ -126,26 +127,15 @@ The easiest way to get started with Zowe is by downloading the convenience build
   </ul>
 </p>
 </details>
-{% if site.cli_active_development_download_url %}
```

Comment:
Please leave this block in the index.md, in case we re-use it in the future. With the blank value for cli_active_development_download_url, it won't appear.